docker-compose-production.yml

```diff
@@ -9,6 +9,12 @@ services:
       dockerfile: docker/prod/django/Dockerfile
     ports:
       - "8000:8000"
+    logging:
+      driver: awslogs
+      options:
+        awslogs-region: ${AWS_DEFAULT_REGION}
+        awslogs-group: django_production
```

Comment:
rest of the `awslogs-group` field are in format `"<log_group>"`. This one isn't, can we keep it consistent for all fields.